docs/service_architecture.md

```diff
@@ -166,6 +169,87 @@ keys in the HSM, but not to exfiltrate the private material.
   signing arbitrary metadata, with the important detail that in this particular
   case key rotations will be necessary to recover from the attack.
 
+### Notary client keys and credentials compromise
+
+In the event of a trust collection owner/administrator's keys being compromised,
+an attacker would be able to sign valid changes to the contents of that
```

Comment:
I notice that you mention the snapshot key compromise below, but it might be better to start with that information up front as well as how the level of severity depends on what kind and how many keys are compromised